pyiron_continuum/schroedinger/schroedinger.py

```diff
@@ -101,7 +104,8 @@ def get_boltzmann_occupation(self, temperature):
     def get_boltzmann_rho(self, temperature):
         if self.psi is not None:
             w = self.get_boltzmann_occupation(temperature)
-            return self._weight_states(self.rho, w).sum(axis=0)
+            weighted_rho = self._weight_states(self.rho, w).sum(axis=0)
+            return weighted_rho / weighted_rho.sum()
```

Comment:
can you double check that this normalization is necessary? IIRC the boltzmann weights are now normalized by their sum (i.e. w/Z) so this should already be normalized.

Reply:
You are right. It is already normalized. I will revert this part.